terraform/preprod.tfvars.json

```diff
@@ -19,5 +19,6 @@
       "node_locations": ["UKINT", "UK1", "MAN1", "MAN5", "DUB2"],
       "confirmations": 2
     }
-  }
+  },
+  "prometheus_app": "prometheus-tra-monitoring-dev"
```

Comment:
Should this be `preprod` rather than `dev` or do you share a prometheus instance across non-prod environments?

Reply:
We only have dev and prod deployments of prometheus instance so yes shared.